Makefile

```diff
@@ -1,36 +1,45 @@
 
 # Default configuration
-SHELL     := /bin/bash
-PYTHON    := python3
-PIP       := ${PYTHON} -m pip
-LINTER    := pycodestyle
-FORMATTER := black
-MAKE_TAG  := --no-print-directory
+SHELL       := /bin/bash
+PYTHON      := python3
+PIP         := ${PYTHON} -m pip
+LINTER      := pycodestyle
+FORMATTER   := black
+MAKE_TAG    := --no-print-directory
 
 # Virtual env
-VENV_NAME = venv
+VENV_NAME   := venv
+VENV_PYTHON := ${VENV_NAME}/bin/${PYTHON}
 
 
-.PHONY: echo-cyan, echo-purple, echo-green, requirements, lint, format, clean
+.PHONY: echo-cyan, echo-purple, echo-green, requirements, migration, serve-back, lint, format, clean
 
 
 ## Create virtualenv
 venv:
-	@$(MAKE) $(MAKE_TAG) echo-cyan msg="Creating and seting up a virtualenv..."
+	@$(MAKE) $(MAKE_TAG) echo-cyan msg="Creating and setting up a virtualenv..."
 	@$(PYTHON) -m venv $(VENV_NAME)
 	@. $(VENV_NAME)/bin/activate; \
 	$(PIP) install --upgrade pip; \
 	$(PIP) install -r requirements.txt
-	@$(MAKE) $(MAKE_TAG) echo-green msg="Virtual created environment successfully ! ✨"
+	@$(MAKE) $(MAKE_TAG) echo-green msg="Virtual environment created successfully ! ✨"
 	@echo
 	@echo "To activate it, please run 'source venv/bin/activate'"
 
 ## Update requirements.txt file
 requirements: venv
-	@rm requirements.txt 2> /dev/null
-	@. venv/bin/activate; \
+	@. $(VENV_NAME)/bin/activate; \
+	$(PIP) install -r requirements.txt; \
+	@rm requirements.txt 2> /dev/null; \
```

Comment:
I don't know why this line is needed. As `pip freeze > ...` overwrites the existing file.

Reply:
<img width="474" alt="Screenshot 2022-11-19 at 10 08 39" src="https://user-images.githubusercontent.com/58826859/202843620-2704d8c3-64fb-4a84-90f9-e169de8bffa4.png">

Reply:
Not working that well on macOS haha

Reply:
Wow
That's why bash >>>> * 😂